### ISSUE TITLE
Document Macros

### ISSUE THREAD

**Issue by mikedotexe** [near/near-sdk-rs#360]
View the current Rust docs here:
https://docs.rs/near-sdk/3.1.0/near_sdk/index.html#macros

- [ ] Provide description for `log`. When you click inside log, provide a description and an example. Something like, "You can easily log with a variable and have the log show up in something like NEAR Explorer to look like this…"
- [ ] Provide description for `testing_env` and show example and/or link to a simple example of this. (Perhaps in this repo for the cross contract high level exmaple)
- [ ] Extra credit for documenting other parts of this module that would benefit developers and might be frequently visited.

In the right sidebar under **Epics**, click on the link for info on running Rust docs locally and good examples of documentation.

**Comment by matklad** [comment]
I'd add 

* [ ] document `near_bindgen`. The docs are empty at the moment https://docs.rs/near-sdk/3.1.0/near_sdk/attr.near_bindgen.html, but that's actually the main macro that makes the contract.

**Comment by mikedotexe** [comment]
> I'd add
> 
> * [ ]  document `near_bindgen`. The docs are empty at the moment https://docs.rs/near-sdk/3.1.0/near_sdk/attr.near_bindgen.html, but that's actually the main macro that makes the contract.

Absolutely, thanks. I was just adding a ticket for that section, good timing. It's here:
https://github.com/near/near-sdk-rs/issues/363

**Comment by austinabell** [comment]
I can take this. Is someone already working on it?
